src/uct/tcp/tcp_iface.c

```diff
@@ -111,6 +111,12 @@ static ucs_config_field_t uct_tcp_iface_config_table[] = {
                 UCS_CONFIG_TYPE_TIME_UNITS},
 #endif /* UCT_TCP_EP_KEEPALIVE */
 
+  {"EP_BIND_SRC_ADDR", "y",
```

Comment:
maybe make it yes/no/try?
so "yes" will cause failure and error messages if cannot bind for some reason, and "try" will just ucs_diag and continue. default will be try.